src/mixins/PreviewUrl.js

```diff
@@ -63,12 +63,17 @@ export default {
 		 * @returns {String} the absolute url
 		 */
 		getPreviewIfAny({ fileid, filename, hasPreview, davPath }) {
+			const searchParams = `fileId=${fileid}`
+				+ `&x=${Math.floor(screen.width * devicePixelRatio)}`
+				+ `&y=${Math.floor(screen.height * devicePixelRatio)}`
+				+ '&a=true'
```

Comment:
```suggestion
			const searchParams = {
				fileId: fileid,
				x: Math.floor(screen.width * devicePixelRatio),
				y: Math.floor(screen.height * devicePixelRatio),
				a: true,
			}
```

Reply:
@skjnldsv , apparently this is not how it works:

> url address (can contain placeholders e.g. /call/{token} would replace {token} with the value of params.token

see: https://github.com/nextcloud/nextcloud-router/blob/442af75e04a74462963cab093fe2c859dc7226a3/lib/index.ts#L91-L102